update sourceComments documentation in README.md

screamingmunch opened the pull request:
expand on sourceComments description in README.md

nschonni commented on README.md:
```diff
@@ -351,7 +351,8 @@ var result = sass.renderSync({
   outFile: '/to/my/output.css',
-  sourceMap: true, // or an absolute or relative (to outFile) path
+  sourceMap: true, // or an absolute or relative (to outFile) path,
+  sourceComments: true,  
```
Not needed since the Sass project doesn't recommend using this option anymore (AFIK)

nschonni commented on README.md:
```diff
@@ -257,7 +257,7 @@ Used to determine how many digits after the decimal will be allowed. For instanc
 Default: `false`
 
-`true` enables additional debugging information in the output file as CSS comments
+`true` enables additional debugging information (line number and source scss file reference, similar to :line_comments in compass) in the output file as CSS comments
```
`:line_numbers` is the Ruby Sass one (with `:line_comments`  as an alias) http://sass-lang.com/documentation/file.SASS_REFERENCE.html#options
Compass is depreciated now, so probably shouldn't be mentioned

xzyfer replied:
Yes please remove the reference to compass.

xzyfer commented on README.md:
```diff
@@ -351,7 +351,7 @@ var result = sass.renderSync({
   outputStyle: 'compressed',
   outFile: '/to/my/output.css',
-  sourceMap: true, // or an absolute or relative (to outFile) path
+  sourceMap: true, // or an absolute or relative (to outFile) path, 
```
Please remove the comma at the end of the comment

xzyfer commented:
Thank you @screamingmunch! ❤️

screamingmunch commented:
Apologies to all the rookie misses.. first-time contributing to open source.  

Removed references to compass, and removed the comma in code sample(left by previous author).  =)

xzyfer commented:
No worries @screamingmunch, we're excited you have your help!
